lib/net.c

```diff
@@ -93,16 +93,10 @@ uint8_t rss_key_be[RTE_DIM(default_rss_key)];
 
 static const struct rte_eth_conf gatekeeper_template_port_conf = {
```

Comment:
`gatekeeper_template_port_conf` is only used once in `init_iface()` and it's so simple now that we are better off just initializing `port_conf`, the local variable in `init_iface()` that receives `gatekeeper_template_port_conf`, with the assignment here.